lib/everlane/asset_upload/manifest_uploader.rb

```diff
@@ -22,6 +22,24 @@ def files
       end
     end
 
+    def manifest_files
+      get_files_from_manifest(@manifest).to_a
+    end
+
+    def get_files_from_manifest(manifest)
+      unique_files = Set.new()
+      manifest.map do |_, val|
+        if val.instance_of?(String)
+          unique_files.add(val)
+        elsif val.instance_of?(Array)
+          val.map{|file| unique_files.add(file) }
+        else
+          unique_files.merge(get_files_recursively(val))
```

Comment:
Is `get_files_recursively` defined? Do you mean `get_files_from_manifest`?

Reply:
i sure did